modape/modis/smooth.py

```diff
@@ -438,8 +438,17 @@ def smooth(self,
 
 
     @property
-    def last_collected(self):
-        """Last collected date in file"""
+    def last_collected_in_raw(self):
+        """Last collected date in raw file"""
+        with h5py.File(self.rawfile, "r") as h5f_raw:
+            dates = h5f_raw.get("dates")
+            last_date = dates[-1].decode()
+
+        return last_date
+
+
+    def last_smoothed(self):
```

Comment:
I guess I'm still not all too sure why we need both `last_collected` and `last_smoothed` ...

If checked after the collection and before smoothing, collect > smooth. 

If checked after smoothing, collect == smooth

Now writing it down I guess I get the sense that looking at the raw dates within the smoothed file makes sure these dates were used for smoothing